src/Core/Geometry/deprecated/TopologicalMesh.inl

```diff
@@ -38,8 +39,8 @@ inline TopologicalMesh::TopologicalMesh( const TriangleMesh& triMesh,
     triMesh.vertexAttribs().for_each_attrib(
         [&triMesh, this, &vprop_float, &vprop_vec2, &vprop_vec3, &vprop_vec4]( const auto& attr ) {
             // skip builtin attribs
-            if ( attr->getName() != std::string( "in_position" ) &&
-                 attr->getName() != std::string( "in_normal" ) ) {
+            if ( attr->getName() != std::string( g_attribName[MeshAttrib::VERTEX_POSITION] ) &&
```

Comment:
It's very bad to perform string comparison here. It could be more efficient to compare the MeshAttrib enumeration here without using name of attribute.

Reply:
Well, I don't see why it's bad and how to fix it. If you have a proposal, feel free to do it.
I think on TopologicalMesh attr are identified by names ... and this functions is costly independently of some string comparisons.